game-core/src/main/java/games/strategy/triplea/ai/AiPoliticalUtils.java

```diff
@@ -84,13 +84,15 @@ private static boolean wantToPerFormActionTowardsWar(final PoliticalActionAttach
   private static boolean goesTowardsWar(final PoliticalActionAttachment nextAction, final PlayerID p0,
       final GameData data) {
     for (final String relationshipChangeString : nextAction.getRelationshipChange()) {
-      final String[] relationshipChange = relationshipChangeString.split(":");
-      final PlayerID p1 = data.getPlayerList().getPlayerId(relationshipChange[0]);
-      final PlayerID p2 = data.getPlayerList().getPlayerId(relationshipChange[1]);
+      final PoliticalActionAttachment.RelationshipChange relationshipChange =
+          PoliticalActionAttachment.parseRelationshipChange(relationshipChangeString);
+      final PlayerID p1 = data.getPlayerList().getPlayerId(relationshipChange.player1Name);
```

Comment:
Thoughts on moving the look up to get `PlayerID` rather than just the `String` name into `PoliticalActionAttachment`? Seems every call to this then does the player look up.

Reply:
Yeah, that should be possible.  It will change this to an instance method, but I don't think that's a big deal.  I'll prototype it tomorrow to see if I run into any issues.